ansys/dpf/core/dpf_operator.py

```diff
@@ -134,30 +135,35 @@ def connect(self, pin, inpt, pin_out=0):
         >>> max_fc_op.inputs.connect(disp_op.outputs)
         >>> max_field = max_fc_op.outputs.field_max()
         >>> max_field.data
-        array([[0.59428386, 0.00201751, 0.0006032 ]])
+        array([[0.59428386, 0.00201751, 0.0006032]])
```

Comment:
I think this is causing build failure.
```suggestion
        array([[0.59428386, 0.00201751, 0.0006032 ]])
```

They're one step ahead of pyaedt and they're testing the docstrings for many of their files.  Here, it was actually expecting the space in the output (I think).